include/classes/Updater/Patch.class.php

```diff
@@ -191,23 +191,23 @@ function applyPatch()
 			foreach($this->backup_files as $backup_file)
 				$backup_files .= $backup_file. '|';
 		
-			$updateInfo = array("backup_files"=>mysql_real_escape_string($backup_files));
+			$updateInfo = array("backup_files"=>mysqli_real_escape_string($db, $backup_files));
```

Comment:
Is `$db` here from the one defined in `vitals.inc.php`. If so, please consider to use the one from `DAO.class.php`.

Reply:
```protected $db;```, I was to trying to know if there are any means of calling it outside its class, apart from using the ```public``` keyword. That is one of the reason, the $addslashes are still being used in ```install folder```, Those are part of what I wanted to ask during our meeting today

Reply:
Thanks, I think that approach is better, will work on it.